test/upgrade_test/fission_objects.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+set -e
+
+ns="f-ns"
+ROOT=$(pwd)
+REPO="docker.io/library"
+STABLE_VERSION=1.12.0
+HELM_VARS="helmVars=repository=docker.io/library,image=fission-bundle,pullPolicy=IfNotPresent,imageTag=latest,fetcher.image=docker.io/library/fetcher,fetcher.imageTag=latest,postInstallReportImage=reporter,preUpgradeChecksImage=preupgradechecks" 
```

Comment:
We should two separate vars. If only one is need we should rename this one.
```
HELM_VARS_PREV_RELEASE
HELM_VARS_LATEST_RELEASE
```

Reply:
We don't use any extra vars for stable installation but I have renamed the existing variable to HELM_VARS_LATEST_RELEASE.